### PR TITLE
Support Python 3.13

### PR DESCRIPTION
**Pull request by jacobtylerwalls**
Declare support for Python 3.13.

### CODE REVIEW

**Review by Pierre-Sassoulas — DISMISSED** [review]

**Review by DanielNoord — APPROVED** [review]

**Comment by jacobtylerwalls** [comment]
I don't know why the tests aren't running.

**Comment by Pierre-Sassoulas** [comment]
(it seems it's running now, but I swear I did nothing, apart from tuning it off and on again)

**Review by DanielNoord — DISMISSED** [review]

**Comment by jacobtylerwalls** [comment]
Blocked by https://github.com/pylint-dev/astroid/pull/2503 and astroid 3.3.2

File: tests/functional/ext/typing/typing_consider_using_alias.py [hunk 1]
```diff
@@ -28,7 +28,6 @@
 var7: typing.Hashable  # [consider-using-alias]
 var8: typing.ContextManager[str]  # [consider-using-alias]
 var9: typing.Pattern[str]  # [consider-using-alias]
-var10: typing.re.Match[str]  # [consider-using-alias]
```
Comment (jacobtylerwalls) [inline tests/functional/ext/typing/typing_consider_using_alias.py:31]: It didn't seem worth it to extract this into its own test file that runs on 3.12 and below only. Let's conserve energy.

**Comment by jacobtylerwalls** [comment]
Towncrier failure should be fixed in next patch release, see https://github.com/twisted/towncrier/issues/632#issuecomment-2268618160.

**Review by Pierre-Sassoulas — APPROVED** [review]
Great work ! And the current 3.13 is a release candidate so we could release 3.13 support right now (before python 3.13 is officially out, nice) !

**Pull request merged**